core/controlplane/config/templates/cloud-config-worker

```diff
@@ -142,7 +142,8 @@ coreos:
         {{end}}--register-node=true \
         {{if .Experimental.Taints}}--register-schedulable=false \
         {{end}}--allow-privileged=true \
-        --pod-manifest-path=/etc/kubernetes/manifests \
+        {{if .Experimental.NodeStatusUpdateFrequency}}--node-status-update-frequency={{.Experimental.NodeStatusUpdateFrequency}}
```

Comment:
@jeremyd Are you sure that you need to set not `experimental.nodeStatusUpdateFrequency` but `worker.nodePools[].nodeStatusUpdateFrequency` for node pools?